src/index.html

```diff
@@ -15,16 +15,21 @@
     <link rel="icon" href='./images/green-apples.png'/>
   </head>
   <body>
+    <section class="whole-filter-section clicked-unhidden"></section>
+    <button class="nav-btn-unhidden clicked-unhidden" type="button" name="button"><img class="unhidden-icon" src="../images/cookbook.png" alt="search button">My Recipes</button>
```

Comment:
LOVE the actual hamburger for the hamburger menu. Fantastic design choice, absolutely amazing. Tickles me pink.